backend/Dev.Dockerfile.celery

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+  unzip \
+  gcc \
+  libffi-dev \
+  musl-dev \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# Copy & Install Python dependencies
+COPY requirements.txt requirements.txt
+COPY requirements-dev.txt requirements-dev.txt
+RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir -r requirements-dev.txt
+
+# Copy application code
+COPY ./app /app
+
+# Run Celery worker with watchdog
+CMD ["watchmedo", "auto-restart", "-d", ".", "-R", "-p", "*.py", "--debug-force-polling", "--", "celery", "-A", "video_extractor.celery", "worker", "--loglevel=info", "-c", "16"]
```

Comment:
### development

After some research I figured that it's convenient to use `watchmedo` command on **development**. But we don't want to use any reloading in production.
This will be installed from the `requirements-dev.txt` file. This way we avoid installing it on production.